index.js

```diff
@@ -63,6 +64,9 @@ AssetsWebpackPlugin.prototype = {
 
       if (self.options.entrypoints) {
         chunks = Object.keys(stats.entrypoints)
+        if (self.options.includeFilesWithoutChunk) {
+          chunks.push('') // push "unamed" chunk
+        }
       } else {
         chunks = Object.keys(stats.assetsByChunkName)
         chunks.push('') // push "unamed" chunk
```

Comment:
Let's fix this one too, since we are fixing things ;-)

```suggestion
        chunks.push('') // push "unnamed" chunk
```